_includes/orchestration/start-kubernetes.md

```diff
@@ -26,6 +26,31 @@ Choose whether you want to orchestrate CockroachDB with Kubernetes using the hos
 
     The process can take a few minutes, so don't move on to the next step until you see a `Creating cluster cockroachdb...done` message and details about your cluster.
 
+{% if page.secure == true %}
```

Comment:
@a-robinson, I added this filter to make sure these RBAC steps don't get pulled into the insecure tutorials.

Reply:
Also made sense to move this content into this `include` file. But it's only showing up for `Hosted GKE`. Is RBAC relevant for the other methods as well?

Reply:
RBAC in general is relevant for other methods. These two steps below are a workaround for GKE limitations and aren't relevant for any other environments.